tasks.rb

```diff
@@ -2505,11 +2505,11 @@ def apply_hpxml_modification(hpxml_file, hpxml)
   if ['base-hvac-undersized-allow-increased-fixed-capacities.xml'].include? hpxml_file
     hpxml.header.allow_increased_fixed_capacities = true
   elsif ['base-schedules-detailed-stochastic.xml'].include? hpxml_file
-    hpxml.header.schedules_filepath = '../../HPXMLtoOpenStudio/resources/schedule_files/stochastic.csv'
+    hpxml.header.schedules_filepaths << '../../HPXMLtoOpenStudio/resources/schedule_files/stochastic.csv'
   elsif ['base-schedules-detailed-stochastic-vacancy.xml'].include? hpxml_file
-    hpxml.header.schedules_filepath = '../../HPXMLtoOpenStudio/resources/schedule_files/stochastic-vacancy.csv'
+    hpxml.header.schedules_filepaths << '../../HPXMLtoOpenStudio/resources/schedule_files/stochastic-vacancy.csv'
   elsif ['base-schedules-detailed-smooth.xml'].include? hpxml_file
-    hpxml.header.schedules_filepath = '../../HPXMLtoOpenStudio/resources/schedule_files/smooth.csv'
+    hpxml.header.schedules_filepaths << '../../HPXMLtoOpenStudio/resources/schedule_files/smooth.csv'
```

Comment:
Hmm. Can we start calling the BuildResSchedule measure for these sample files so we don't have to manually modify the HPXML object?

Reply:
This is tough. The `SchedulesFilePath` is relative the `workflow/sample_files` but it is also the path for exporting the schedules, relative `BuildResidentialScheduleFile/measure.rb`.

Reply:
So, we could use an absolute path, but this would require that the sample files get updated on the ci...

Reply:
I don't follow what you mean by "but it is also the path for exporting the schedules, relative BuildResidentialScheduleFile/measure.rb". We definitely don't want to use an absolute path -- what works on your machine won't work on my machine.